modules/dfpAdServerVideo.js

```diff
@@ -101,6 +101,13 @@ export function buildDfpVideoUrl(options) {
   const descriptionUrl = getDescriptionUrl(bid, options, 'params');
   if (descriptionUrl) { queryParams.description_url = descriptionUrl; }
 
+  const gdprConsent = gdprDataHandler.getConsentData();
+  if (gdprConsent) {
+    if (typeof gdprConsent.gdprApplies === 'boolean') { queryParams.gdpr = gdprConsent.gdprApplies; }
```

Comment:
To be more explicit, should gdprConsent.gdprApplies be wrapped in a Number()? the [dfp tcf2 docs](https://support.google.com/admanager/answer/9805023?hl=en) specify 'gdpr' as an integer value

Reply:
I've updated my code. Thanks for pointing that out.